src/lib/githubAPI.ts

```diff
@@ -0,0 +1,20 @@
+async function getOpensaucedGoalsReposCount (): Promise<number> {
```

Comment:
disable this please, it's not scalable since we are already hitting the github client rate limit; if you need this information instead get the total number of repos from the api with 1/1 offsets like so:

```
curl -X 'GET' \
  'https://api.opensauced.pizza/v1/repos/list?page=1&limit=1' \
  -H 'accept: application/json'
```

This will give you the minimum results along with the meta property that holds a lot more information!

Reply:
![image](https://user-images.githubusercontent.com/52013393/205276577-cad19f0f-e4eb-4c69-b95b-5f80df3dcd15.png)
This meta object? It doesn't have the number I'm looking for, which is how many users have forked the open-sauced tracker repo.